Extension/package.nls.zh-cn.json

```diff
@@ -1,7 +1,8 @@
 {
     "c_cpp.command.configurationSelect.title": "选择配置...",
     "c_cpp.command.configurationProviderSelect.title": "更换配置提供程序...",
-    "c_cpp.command.configurationEdit.title": "编辑配置...",
+    "c_cpp.command.configurationEditJSON.title": "编辑配置... (JSON)",
+    "c_cpp.command.configurationEditUI.title": "编辑配置... (UI)",
```

Comment:
does vscode translate "UI" in their command?

Reply:
VS code doesn't translate UI in the Chinese language pack.